tools/force-shading/pom.xml

```diff
@@ -93,11 +93,13 @@ under the License.
 				<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-release-plugin</artifactId>
-					<version>2.1</version>
+					<version>2.5.3</version>
 					<configuration>
-						<mavenExecutorId>forked-path</mavenExecutorId>
-						<useReleaseProfile>false</useReleaseProfile>
-						<arguments>${arguments} -Psonatype-oss-release</arguments>
+						<releaseProfiles>release</releaseProfiles>
+						<preparationGoals>clean install</preparationGoals>
+						<goals>deploy</goals>
+						<autoVersionSubmodules>true</autoVersionSubmodules>
+						<pushChanges>false</pushChanges>
```

Comment:
`autoVersionSubmodules` tells it to update the version in all the submodules. `pushChanges` instructs it to not early push new branches, commits, tags to git. Otherwise it does that, which seems a bit crazy.